cypress/integration/headerSpec.js

```diff
@@ -0,0 +1,39 @@
+import {
+  getElement,
+  shouldContainText,
+  shouldContainStyles,
+} from '../support/testHelper';
+
+describe('Header Tests', () => {
+  // eslint-disable-next-line no-undef
+  before(() => {
+    cy.visit('/article/scenario-25');
+  });
+
+  it('should render the BBC News branding', () => {
```

Comment:
Now that this is split out, it would be of value to split out the specific assertions into their own test, rather than a generic `should render`.